### ISSUE TITLE
Program Gap Tech

### ISSUE THREAD

**Issue by ariya** [deeptechid/deeptech.id#222]
Sudah dipromosikan di medsos, tapi belum dicantumkan di deeptech.id.

**Comment by ramarahmanda** [comment]
Yup sudah dibahas mas, kita akan tambahkan gaptech dan beberapa program lainnya ke web.
